### PR TITLE
https://issues.redhat.com/browse/ACM-6292 One way limitations 2.9

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.8.

### CODE REVIEW

File: clusters/release_notes/known_issues.adoc [hunk 1]
```diff
@@ -416,7 +416,21 @@ oc annotate --overwrite AgentServiceConfig agent unsupported.agent-install.opens
...
 
- 
+[#hub-cluster-one-way-limits]
+=== Hub cluster communication limitations
+//2.9:ACM-6292
+
+The following limitations occur if the hub cluster is not able to reach or communicate with the managed cluster:
+
+- You cannot create a new managed cluster by using the console. You are still able to import a managed cluster manually by using the command line interface or by using the *Run import commands manually* option in the console.
+- If you deploy an Application or ApplicationSet by using the console, or if you import a managed cluster into ArgoCD, the hub cluster ArgoCD controller calls the managed cluster API server. You can use AppSub or ArgoCD pull model to work around the issue.
```
Comment (cadawson) [inline clusters/release_notes/known_issues.adoc]: "You can use the AppSub or ArgoCD..." ? Or maybe "You can use AppSub or the ArgoCD pull model"? Added "the."

File: clusters/release_notes/known_issues.adoc [hunk 2]
```diff
@@ -416,7 +416,21 @@ oc annotate --overwrite AgentServiceConfig agent unsupported.agent-install.opens
...
+- The console page for pod logs does not work and an error message that resembles the following appears:
```
Comment (cadawson) [inline clusters/release_notes/known_issues.adoc]:
```suggestion
- The console page for pod logs does not work, and an error message that resembles the following appears:
```

**Review by cadawson — APPROVED** [review]
Looks good. A couple of suggestions. Thanks!

File: clusters/release_notes/known_issues.adoc [hunk 3]
```diff
@@ -416,7 +416,21 @@ oc annotate --overwrite AgentServiceConfig agent unsupported.agent-install.opens
...
 The `selectorType: LaberSelector` setting is not supported when using the link:../../apis/managedclusterset.json.adoc#managedclusterset-api[ManagedClusterSet API]. The `selectorType: ExclusiveClusterSetLabel` setting is supported.
 
- 
+[#hub-cluster-one-way-limits]
+=== Hub cluster communication limitations
+//2.9:ACM-6292
+
+The following limitations occur if the hub cluster is not able to reach or communicate with the managed cluster:
+
+- You cannot create a new managed cluster by using the console. You are still able to import a managed cluster manually by using the command line interface or by using the *Run import commands manually* option in the console.
```
Comment (zhujian7) [inline clusters/release_notes/known_issues.adoc:425]: lgtm for the cluster importing part.

**Pull request merged**